### DISCUSSION TITLE
Force fetch remote content

### DISCUSSION THREAD

**Discussion by skid9000** [misskey-dev/misskey#9204] [Q&A]
Hello,

is there a way to force misskey to fetch old content from a remote account ?

I have a remote mastodon account reported with 1 status and created 20h ago (but exists since 2017) and the "update remote user information button" isn't doing anything that i can see.

**Comment by ThatOneCalculator** [comment]
Backfilling isnt a thing, but you can copy the note url search for it

**Reply by Trif4** [reply]
It makes sense that you wouldn't want to copy 10k old notes from the remote instance, but would it be possible for the client to display them retrieved directly from the remote?

**Reply by skid9000** [reply]
Or maybe fetch a defined number of old notes so that the profile isn't empty ?

**Comment by EpicLPer** [comment]
This...

I've set up my own instance, followed my other account just to then notice it doesn't load old posts. That is a huge dealbreaker for me cause, from the time on of creating my instance it means I'm missing 100% of the people's posts and can't scroll back. I'd love for there to be an option to automatically fetch remote profiles in a way that doesn't essentially DDoS other servers so I can view what they posted before without having to manually copy/paste URLs around.
